src/blocks/apt.rs

```diff
@@ -189,11 +203,18 @@ async fn get_updates_list(config_path: &str) -> Result<String> {
 async fn get_update_count(
     config_path: &str,
     ignore_phased_updates: bool,
+    ignore_updates_regex: Option<&Regex>,
     updates: &str,
 ) -> Result<usize> {
     let mut cnt = 0;
 
-    for update_line in updates.lines().filter(|line| line.contains("[upgradable")) {
+    for update_line in updates
+        .lines()
+        .filter(|line| line.contains("[upgradable"))
+        .filter(|line| {
+            ignore_updates_regex.is_none() || !ignore_updates_regex.unwrap().is_match(line)
```

Comment:
This can be written as `ignore_updates_regex.map_or(true, |re| !re.is_match(line))`. Sad that `.is_none_or(...)` which would be even better [was rejected](https://github.com/rust-lang/rust/pull/100602#issuecomment-1554422512).

Reply:
Done

Reply:
@MaxVerevkin, is there anything else required for me to do to get this merged?

Reply:
No, everything looks good to me :)